src/hammer-vlsi/defaults.yml

```diff
@@ -158,6 +158,15 @@ vlsi.inputs:
   # - "power" - This obstruction type stops only power routing/straps inside it
   placement_constraints: []
 
+vlsi.submit:
+  # The submit command to use. "none", "local", or null will run on the current host. See hammer_submit_command.py for other options.
+  command: "none"
```

Comment:
`none` sounds deceptive here since it might imply that no command gets run. Maybe let's just leave it at `local`?

Reply:
Agree.